corehq/apps/accounting/tasks.py

```diff
@@ -623,13 +623,13 @@ def _fmt_row(sub):
         })
 
 
-@periodic_task(serializer='pickle', run_every=crontab(hour=1, minute=0,), acks_late=True)
+@periodic_task(run_every=crontab(hour=1, minute=0,), acks_late=True)
 def pay_autopay_invoices():
     """ Check for autopayable invoices every day and pay them """
     AutoPayInvoicePaymentHandler().pay_autopayable_invoices(datetime.datetime.today())
 
 
-@periodic_task(serializer='pickle', run_every=crontab(minute=0, hour=0), queue='background_queue', acks_late=True)
+@periodic_task(run_every=crontab(minute=0, hour=0), queue='background_queue', acks_late=True)
 def update_exchange_rates(app_id=settings.OPEN_EXCHANGE_RATES_API_ID):
```

Comment:
This function is only ever queued by celery beat - hence no arguments are passed.